src/pa02/test_pa02.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Trude Haug Almestrand', 'Nina Mariann Vesseltun'
+__email__ = 'Trude.haug.almestrand@nmbu.no', 'nive@nmbu.no'
+
+import src.pa02.snakes_simulation as cs
+import random
+
+
+def test_standard_board():
+    """Checking if standard board created is correct"""
+    b = cs.Board()
+    assert b.chutes == [(24, 5), (33, 3), (42, 30), (56, 37), (64, 27),
+                        (74, 12), (87, 70)]
+    assert b.ladders == [(1, 40), (8, 10), (36, 52), (43, 62), (49, 79),
+                         (65, 82), (68, 85)]
+    assert b.goal == 90
+
+
+def test_goal_reached():
+    """Checking if goal_reached works"""
+    b = cs.Board()
+    assert b.goal_reached(90) is True
+    assert b.goal_reached(13) is False
+    assert b.goal_reached(100) is True
```

Comment:
Please have the habit of omitting `is True` and using `not` instead of `is False`